strategies/AbstractStrategy.py

```diff
@@ -27,8 +27,11 @@ def simulate(
         chosen_iter, total_iter = self._run(stopping_curve)
         opt_iter = eval_curve.index(min(eval_curve))
         error_diff = eval_curve[chosen_iter] - eval_curve[opt_iter]
-        max_eval = max(eval_curve)
-        percent_error_diff = error_diff / (max_eval if max_eval != 0 else 1)
+        assert error_diff >= 0
+        if eval_curve[opt_iter] == eval_curve[chosen_iter]:
+            percent_error_diff = 0
+        else:
+            percent_error_diff = 1 - (eval_curve[opt_iter] / (eval_curve[chosen_iter] if eval_curve[chosen_iter] != 0 else 1))
```

Comment:
If the optimal error was the denominator, then the value could be infinity and would always be >=1.

I instead want to calculate something different. Specifically I want the % error reduction of the optimal compared to the chosen. This will always be <=1 and >=0, bounding the value.

Reply:
Ah, i see now. Noted.